gen/build_deploy/bash.py

```diff
@@ -136,9 +136,9 @@
 
 # check if sort -V works
 function check_sort_capability() {
-    $( echo '1' | sort -V >/dev/null 2>&1 || exit 1 )
+    $( echo '1' | sort -V >/dev/null 2>&1 )
     RC=$?
-    if [[ "$RC" -eq "2" ]]; then
+    if [[ "$RC" -eq "0" ]]; then
```

Comment:
Actually I just took a look at the docs for GNU sort: https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html
> Exit status:
>    0 if no error occurred
>    1 if invoked with -c or -C and the input is not sorted
>    2 if an error occurred

An exit status of 1 doesn't indicate an error, so let's stick with the `-eq "2"`.